azure-iot-device/azure/iot/device/provisioning/provisioning_device_client.py

```diff
@@ -58,6 +61,10 @@ def register(self):
 
     def cancel(self):
```

Comment:
>cancel [](start = 8, length = 6)

Out of scope, but we should consider the need to include this API before we GA.  I know that we have this in the Node SDK, but, AFAIK, this is the only operation that we have in the entire Python SDK that is cancelable.  I would question the utility of this function.  If we do need it, we need to make sure we know why, and we need to make sure it's bulletproof

Reply:
I agree

---
In reply to: [334947310](https://github.com/Azure/azure-iot-sdk-python/pull/349#discussion_r334947310) [](ancestors = 334947310)